pom.xml

```diff
@@ -37,6 +37,7 @@
         <module>integrationtest/wildfly</module>
         <module>integrationtest/performance</module>
         <module>integrationtest/osgi/karaf-features</module>
+        <module>integrationtest/osgi/karaf-it</module>
```

Comment:
Ah, I see. I thought it'd be missing, indeed didn't see the profile
(wouldn't it make sense to add the actual feature definition to that
profile then as well; seems less confusing). So feel free to remove that
one.